src/content/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/automatic-anomalies.mdx

```diff
@@ -17,10 +28,792 @@ Automatic anomaly detection is the easiest way for you and your team to detect a
 
 **Anomaly analysis:** For each anomaly, we provide a link in Slack to an analyze anomaly page. This page generates automatic insights into the anomaly. The page is also available from the anomalies tab, which lists recent anomalies. This page uses your existing <InlinePopover type="apm" /> and anomaly detection data to provide explanations as to the cause of the anomaly.
 
 **Activity stream:** Inside various activity streams such as the New Relic homepage, <InlinePopover type="apm" /> summary page, Lookout, and Explorer, you'll see relevant anomalies from your APM-monitored applications. Clicking on any of the anomaly events in the activity stream brings up the analysis page for that anomaly.
 
 Applications don't always generate anomalies, so it can be normal to not receive any detections.
+
+
+## Configure anomaly detection [#conf-anomaly-detection]
+
+To configure an anomaly detection:
+
+1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Alerts & AI** and click **Anomaly detection** in the left pane.
+
+2. Click **+ Add a configuration**.
+
+<img
+  title="Configure anomaly detection"
+  alt="Configure anomaly detection"
+  src={anomalyConfAnomalyDetection}
+/>
+<figcaption>
+  To see configure an anomaly detection, go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Alerts & AI** and click **Anomaly detection** in the left pane. 
+</figcaption>
+
+3. Choose a name for your configuration that helps you easily identify it from others in your account.
+
+4. Select the account you want to use for this configuration.
+
+5. Select the applications and services you want to include.
+
+6. Select the metrics you want to be notified on.
+
+7. Select how you want to be notified. These are the options:
+
+    * **Slack**. You need to select a Slack channel and the time zone the charts will use.
+    * **Webhook**. You need to add the URL.
+    * **No notifications**. If you prefer not to receive notifications, select this option.
+
+8. Click **Save configuration**.
+
+
+## How to see the details of an anomaly [#details-anomaly]
+
+You can explore the details of any anomaly in your system's performance to better understand what errors you're receiving and why.
+
+<img
+  title="A screenshot of anomaly details in the New Relic UI"
+  alt="A screenshot of anomaly details in the New Relic UI"
+  src={anomalyDetails}
+/>
+<figcaption>
+  To see the details of a detected anomaly, go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Alerts & AI**. Click the **Anomalies** tab to see the dashboard and then click the anomaly you want to see. 
+</figcaption>
+
+The details of an anomaly gives you this information:
+
+  * Name of the anomaly. You can click on it and it will take you to APM & Services capability to see a summary of it. 
```

Comment:
```suggestion
  * Name of the anomaly. You can click on it and it will take you to APM & Services capability to see a summary of the selected anomaly.
```